src/main/java/com/novi/backend_spring_boot_tech_it_easy_controller/models/Television.java

```diff
@@ -0,0 +1,211 @@
+package com.novi.backend_spring_boot_tech_it_easy_controller.models;
+
+import jakarta.persistence.*;
+
+@Entity
+public class Television {
+
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    private String type;
+    @Column(nullable = false)
+    private String brand;
+    @Column(nullable = false)
+    private String name;
+    @Column(nullable = false)
+    private Double price;
+
+    private Double availableSize;
+    private Integer refreshRate;
+    private String screenType;
+    private String screenQuality;
+    private Boolean smartTv;
+    private Boolean wifi;
+    private Boolean voiceControl;
+    private Boolean hdr;
+    private Boolean bluetooth;
+    private Boolean ambiLight;
+    private Integer originalStock;
+    private Integer sold;
+
+
+
+
+    public Television() {
+
+    }
+
+
+public Television(Long id, String type, String brand, String name, Double price, Double availableSize, Integer refreshRate, String screenType, String screenQuality, Boolean smartTv, Boolean wifi, Boolean voiceControl, Boolean hdr, Boolean bluetooth, Boolean ambiLight, Integer originalStock, Integer sold){
+
+        this.id = id;
+        this.type = type;
+        this.brand = brand;
+        this.name = name;
+        this.price = price;
+        this.availableSize = availableSize;
+        this.refreshRate = refreshRate;
+        this.screenType = screenType;
+        this.screenQuality = screenQuality;
+        this.smartTv = smartTv;
+        this.wifi = wifi;
+        this.voiceControl = voiceControl;
+        this.hdr = hdr;
+        this.bluetooth = bluetooth;
+        this.ambiLight = ambiLight;
+        this.originalStock = originalStock;
+        this.sold = sold;
+}
+
+    @OneToOne(cascade = CascadeType.ALL)
+    @JoinColumn(name = "remote_controller_id", referencedColumnName = "id")
```

Comment:
In jou dataabse heeft de Television tabel nu een extra kolom gekregen genaamd "remote_controller_id" die verwijst naar de "id" kolom van de RemoteController tabel